velox/docs/functions/spark/array.rst

```diff
@@ -94,6 +94,7 @@ Array Functions
         SELECT array_max(array(-1, -2, NULL)); -- -1
         SELECT array_max(array()); -- NULL
         SELECT array_max(array(-0.0001, -0.0002, -0.0003, float('nan'))); -- NaN
+        SELECT array_max(array(array(1), array(NULL))) -- array(1)
```

Comment:
Would you add description on how nested element is ordered, e.g. when the arrays contain NULL and NAN, and how arrays of different size are compared?

Reply:
Examples are added. Thanks.